cdnetworks/actions.py

```diff
@@ -4,11 +4,7 @@ def __init__(self, api_base, session_token, api_key):
         self.session_token = session_token
         self.api_key = api_key
 
-    def create_pad(self, contract_number, new_pad_name, origin, description=None):
```

Comment:
Why did you remove create_pad function?

Reply:
it wasn't used by anywhere in the project